charts/renovate/templates/cronjob.yaml

```
@@ -185,6 +211,8 @@ spec:
             {{- end }}
             {{- if .Values.ssh_config.enabled }}
             - name: ssh-config-volume
+              emptyDir: {}
```

```suggestion
              emptyDir:
                medium: Memory
```
We should only store it in memory for security